tests/Integrations/Hubspot/HubspotSettingsTest.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Tests\Unit\Integrations\Hubspot;
+
+use Brain\Monkey;
+use EightshiftForms\Integrations\ClientInterface;
+use EightshiftForms\Integrations\Hubspot\Hubspot;
+use EightshiftForms\Integrations\Hubspot\HubspotClient;
+use EightshiftForms\Integrations\Hubspot\SettingsHubspot;
+use EightshiftForms\Integrations\MapperInterface;
+use EightshiftForms\Labels\Labels;
+use EightshiftForms\Validation\Validator;
+
+use function Tests\setupMocks;
+
+class SettingsHubspotMock extends SettingsHubspot {
+
+	public function __construct(
+		ClientInterface $hubspotClient,
+		MapperInterface $hubspot
+	) {
+		parent::__construct($hubspotClient, $hubspot);
+	}
+};
+
+/**
+ * Mock before tests.
+ */
+beforeEach(function () {
+	Monkey\setUp();
+	setupMocks();
+
+	$hubspotClient = new HubspotClient();
+	$labels = new Labels();
+	$validator = new Validator($labels);
+	$hubspot = new Hubspot($hubspotClient, $validator);
+
+	$this->hubspotSettings = new SettingsHubspotMock($hubspotClient, $hubspot);
```

Comment:
Why not use mockery instead of creating a class here on line 16?

Reply:
sorry I wasn't able to do it that way :(